### PR TITLE
DB-10745 fix TriggerInfo serialization

### PR DESCRIPTION
**Pull request by martinrupp**

### CODE REVIEW

File: splice_machine/src/main/java/com/splicemachine/derby/impl/store/access/hbase/HBaseConglomerate.java [hunk 1]
```diff
@@ -353,8 +353,10 @@ private void localReadExternal(ObjectInput in)
         num_columns=in.readInt();
         columnOrdering=ConglomerateUtil.readFormatIdArray(num_columns,in);
 
-        partitionFactory=SIDriver.driver().getTableFactory();
-        opFactory=SIDriver.driver().getOperationFactory();
+        if( SIDriver.driver() != null ) {
```
Comment (martinrupp) [inline splice_machine/src/main/java/com/splicemachine/derby/impl/store/access/hbase/HBaseConglomerate.java:356]: in the serialization tests we don't have the driver

**Review by dgomezferro — APPROVED** [review]

**Review by msirek — APPROVED** [review]
Nice work!

**Review by carolp-503 — APPROVED** [review]
+1

**Comment by martinrupp** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by spliceci** [comment]
TEST FAILED
http://nexus.splicemachine.com:8080/job/spliceengine-PR/job/Splice-PR-Branch/1959/

**Comment by martinrupp** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by spliceci** [comment]
TEST FAILED
http://nexus.splicemachine.com:8080/job/spliceengine-PR/job/Splice-PR-Branch/1961/

**Comment by martinrupp** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by spliceci** [comment]
TEST FAILED
http://nexus.splicemachine.com:8080/job/spliceengine-PR/job/Splice-PR-Branch/1968/

**Review by carolp-503 — COMMENTED** [review]
+1

**Pull request merged**